tests/tests/constants/index.ts

```diff
@@ -0,0 +1,68 @@
+export {
+  TEST_CONTRACT_BYTECODE,
+  TEST_CONTRACT_ABI,
+  FIRST_CONTRACT_ADDRESS,
+  TEST_CONTRACT_BYTECODE_INCR,
+  TEST_CONTRACT_INCR_ABI,
+  INFINITE_CONTRACT_BYTECODE,
+  INFINITE_CONTRACT_ABI,
+  INFINITE_CONTRACT_BYTECODE_VAR,
+  INFINITE_CONTRACT_ABI_VAR,
+  FINITE_LOOP_CONTRACT_BYTECODE,
+  FINITE_LOOP_CONTRACT_ABI,
+} from "./testContracts";
+
+export { basicTransfertx, contractCreation } from "./transactionConfigs";
+
+export const PORT = 19931;
```

Comment:
I'm hesitant to use those global to the test as at some point we will have different values needed, (specially when testing multiple accounts and ...).
I guess it is fine to keep it for now and we will review it once needed